src/runtime_src/core/tools/common/ReportCmcStatus.cpp

```diff
@@ -115,34 +116,35 @@ ReportCmcStatus::writeReport( const xrt_core::device* /*_pDevice*/,
   }
 
   try {
-    boost::property_tree::ptree cmc_scale = cmc.get_child("scaling");
-    _output << boost::format("  %-22s:\n") % cmc_scale.get<std::string>("Description");
-    if (!cmc_scale.get<bool>("supported")) {
-      _output << "    Not supported\n";
+    boost::property_tree::ptree cmc_throttle = cmc.get_child("throttling");
+    _output << boost::format("  %-22s:\n") % cmc_throttle.get<std::string>("Description");
+
+    if (!cmc_throttle.get<bool>("supported")) {
+      _output << "    Not supported. Ensure xclbin is loaded.\n";
       return;
     }
-    if (!cmc_scale.get<bool>("enabled")) {
+    if (!cmc_throttle.get<bool>("enabled")) {
```

Comment:
Add a new line to separate the if statements. I thought they were an if/elseif grouping!